pkg/controller/bootstrap/jws.go

```diff
@@ -34,17 +34,17 @@ func computeDetachedSig(content, tokenID, tokenSecret string) (string, error) {
 
 	signer, err := jose.NewSigner(jose.HS256, jwk)
 	if err != nil {
-		return "", nil
+		return "", fmt.Errorf("can't make a HS256 signer from the given token")
```

Comment:
Just curious why create our own error and not use the one we've been given?

I would assume the one we're given would have more debuggable information. Looking at the current version of `jose-go` the `NewSigner` method for example should only return an error if it doesn't support that algorithm. 

Wouldn't that be more helpful? Otherwise we are being told that a signer can't be created for the token, which would make me think that I've used a malformed token.